src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/GrapplingHookEntity.java

```diff
@@ -15,13 +15,14 @@
 final class GrapplingHookEntity {
 
     private final boolean dropItem;
-
+    private final boolean keepininv;
```

Comment:
Pascal case, so `keepInInv`, or with the suggested reversal: `wasConsumed` may be more fitting.